app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  has_many :comments
```

Comment:
When you define has_many associations, please use `dependent: :destroy`. When you delete a post, comments should be deleted, yeah?